README.md

```diff
@@ -1,41 +1,34 @@
-<p align="center">
-  <img width="128" align="center" src="assets/fluenthub.png" />
-  <img width="128" align="center" src="assets/fluenthub-canary.png" />
-  <img width="128" align="center" src="assets/fluenthub-dev.png" />
-  <img width="128" align="center" src="assets/fluenthub-beta.png" />
-</p>
-<h1 align="center">
-  FluentHub
-</h1>
-<p align="center">
-  A stylish yet powerful GitHub client.
-</p>
+# <img width="64" align="center" src="assets/fluenthub.png" /> <img width="64" align="center" src="assets/fluenthub-dev.png" /> FluentHub
+
+###### A stylish yet powerful GitHub client
 
 <p align="center">
   <a title="Azure Pipeline" target="_blank" href="https://dev.azure.com/fluenthub/FluentHub">
-    <img src="https://dev.azure.com/fluenthub/FluentHub/_apis/build/status/Build%20Pipeline%20(x64)?branchName=main">
+    <img align="left" src="https://dev.azure.com/fluenthub/FluentHub/_apis/build/status/Build%20Pipeline%20(x64)?branchName=main">
   </a>
   <a title="Crowdin" target="_blank" href="https://crowdin.com/project/fluenthub">
-    <img src="https://badges.crowdin.net/fluenthub/localized.svg">
+    <img align="left" src="https://badges.crowdin.net/fluenthub/localized.svg">
   </a>
   <a title="GitHub Releases" target="_blank" href="https://github.com/fluenthub-community/FluentHub/releases">
-    <img src="https://img.shields.io/github/v/release/fluenthub-community/fluenthub?include_prereleases" alt="Release" />
+    <img align="left" src="https://img.shields.io/github/v/release/fluenthub-community/fluenthub?include_prereleases" alt="Release" />
   </a>
   <a title="Discord" target="_blank" href="https://discord.gg/8KtRkjq2Q4">
-    <img src="https://img.shields.io/discord/935562861701390336?color=blue&label=Discord" alt="Discord" />
+    <img align="left" src="https://img.shields.io/discord/935562861701390336?color=blue&label=Discord" alt="Discord" />
   </a>
   <a title="Platform" target="_blank">
-    <img src="https://img.shields.io/badge/Platform-Windows-red" alt="Platform" />
+    <img align="left" src="https://img.shields.io/badge/Platform-Windows-red" alt="Platform" />
   </a>
 </p>
 
+<br/>
+
 ---
 
-## Installation
+## ➕ Installation
 
-### Via Microsoft Store
+##### Via Microsoft Store
```

Comment:
Only phrases and emoji additions are allowed among the changed README.md diff.

- Not allowed:
  - Add `##` before header 3nd.

Reply:
Finished!